libs/community/langchain_community/document_loaders/unstructured.py

```diff
@@ -248,51 +208,119 @@ class UnstructuredAPIFileLoader(UnstructuredFileLoader):
 
     References
     ----------
-    https://unstructured-io.github.io/unstructured/bricks.html#partition
-    https://www.unstructured.io/api-key/
-    https://github.com/Unstructured-IO/unstructured-api
+    https://docs.unstructured.io/api-reference/api-services/sdk
+    https://docs.unstructured.io/api-reference/api-services/overview
+    https://docs.unstructured.io/open-source/core-functionality/partitioning
+    https://docs.unstructured.io/open-source/core-functionality/chunking
     """
 
     def __init__(
         self,
-        file_path: Union[str, List[str], None] = "",
+        file_path: Union[str, list[str]],
+        *,
         mode: str = "single",
         url: str = "https://api.unstructured.io/general/v0/general",
         api_key: str = "",
         **unstructured_kwargs: Any,
     ):
         """Initialize with file path."""
 
-        validate_unstructured_version(min_unstructured_version="0.10.15")
-
+        self.file_path = file_path
         self.url = url
         self.api_key = api_key
 
-        super().__init__(file_path=file_path, mode=mode, **unstructured_kwargs)
```

Comment:
Why did you need to make this change?

Reply:
I had to refactor the classes a bit since the API Loaders were not inheriting directly from the BaseUnstructuredLoader and needed to implement some of their methods themselves. This was removed in that process.